suite-native/module-trading/src/hooks/history/useChangeStringsExtractor.ts

```diff
@@ -38,32 +32,16 @@ export const useChangeStringsExtractor = (
         if (value === undefined || cryptoId === undefined) {
             return undefined;
         }
-        const { coinSymbol } = cryptoIdToSymbolAndContractAddress(cryptoId);
-
-        let formattedValue;
-        if (coinSymbol && isNetworkSymbol(coinSymbol)) {
-            const { decimals } = getNetwork(coinSymbol);
-
-            formattedValue = CryptoAmountFormatter.format(value, {
-                maxDisplayedDecimals: decimals,
-                isBalance: true,
-                symbol: coinSymbol,
-                isEllipsisAppended: false,
-                smallestUnitsOverride,
-            });
-
-            const splitValue = formattedValue.split(' ');
-            if (splitValue.length > 1) {
-                formattedValue = `${formatNumberWithThousandCommas(splitValue[0])} ${splitValue.slice(1).join(' ')}`;
-            } else if (splitValue.length > 0) {
-                formattedValue = formatNumberWithThousandCommas(splitValue[0]);
-            }
-        } else {
-            const decimalValue = convertTokenValueToDecimal(value, 0);
-            formattedValue = `${localizeNumber(decimalValue)} ${coinSymbol?.toLocaleUpperCase() || ''}`;
-        }
-
-        return formattedValue;
+        const coinSymbol = cryptoIdToCoinSymbol(cryptoId);
```

Comment:
Shouldn't `cryptoIdToCoinSymbol` return `NetworkSymbol | undefined`? I think type casting below is unnecessary and should be handled by this function. Understand if it's out of scope for this PR and would be too much refactoring.